rust_src/src/windows.rs

```diff
@@ -697,4 +697,12 @@ pub fn window_list_one(
     unsafe { (window_list_1(window.to_raw(), minibuf.to_raw(), all_frames.to_raw())) }
 }
 
+/// Return old value of point in WINDOW.
+/// WINDOW must be a live window and defaults to the selected one.
+#[lisp_fn(min = "0")]
+pub fn window_old_point(window: LispObject) -> Option<EmacsInt> {
+    let old_pointm = window_live_or_selected(window).old_pointm;
+    marker_position_lisp(old_pointm.into())
```

Comment:
I'd rather see this written as

    let win = window_live_or_selected(window)
    marker_position_lisp(win.old_pointm.into())

This will help debugging.

Reply:
Fixed already.